### DISCUSSION TITLE
GitHub Education Benefits Still Not Activated After Approva

### DISCUSSION THREAD

**Discussion by AJ69swaroop** [community/community#147697] [GitHub Education]
### Hi everyone,

I received confirmation that my GitHub Education request for [@AJ69swaroop](https://github.com/AJ69swaroop) was approved on December 19, 2024. However, it's been three days since the approval, and my benefits are still not activated.

Here are the details:

    Date of Approval: December 19, 2024
    Message Received: "Congrats! Your request for @AJ69swaroop was approved, and your benefits will be available soon."
    
![image](https://github.com/user-attachments/assets/329209b4-e524-4022-acf6-a0c19a20abe2)

I understand there may be some processing time, but I wanted to check if this delay is normal or if I need to take any additional steps to activate my benefits.

Any guidance or assistance would be greatly appreciated.

Thank you!

**Comment by davevad93** [comment]
Hi @AJ69swaroop ,

Yes, the delay is normal. When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AJ69swaroop was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by AJ69swaroop** [reply]
thank you for the reply

**Reply by davevad93** [reply]
You're welcome @AJ69swaroop 👍
